charts/zeebe-benchmark/Chart.yaml

```diff
@@ -10,7 +10,7 @@ sources:
 dependencies:
   - name: camunda-platform
     repository: "oci://ghcr.io/camunda/helm"
-    version: "0.0.0-8.7.0-alpha1"
+    version: "0.0.0-snapshot-alpha"
```

Comment:
⚠️ This is now a snapshot version. When alpha2 is released we can pin this to `0.0.0-8.7.0-alpha2`. For now we need this to be able to use the fixes we made in the camunda platform helm charts.